tools/fuzzer/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ add_executable(sof-fuzzer
 	topology.c
```

Comment:
@crazoes commit message should tell more about the code changes. You can mention at the bottom about the timeout. But if you read the code you will see that most of it is not about this timeout.

Reply:
Okay, I borrowed the commit message from diana's commit and added to it for the changes I made. Will amend.

Reply:
I've changed, mostly describing about MU because other than that we've just implemented the fuzzer API.